Runtime/CesiumTileExcluder.cs

```diff
@@ -0,0 +1,28 @@
+﻿using UnityEngine;
+
+namespace CesiumForUnity
+{
+    [ExecuteInEditMode]
+    public abstract partial class CesiumTileExcluder : MonoBehaviour
+    {
+        internal Cesium3DTile _tile = null;
+
+        /// <summary>
+        /// Determines whether the given tile should be excluded from loading and rendering.
+        /// </summary>
+        /// <param name="tile">The tile to check. This instance is only valid for the duration of this call.</param>
+        /// <returns>True if the tile should be excluded, false if the tile should be loaded and rendered.</returns>
+        public abstract bool ShouldExclude(Cesium3DTile tile);
+
+        protected virtual void OnEnable()
+        {
+            this._tile = new Cesium3DTile();
+            this._tile._georeference = GetComponentInParent<CesiumGeoreference>();
```

Comment:
This code has been removed, but I've changed the C++ code so that if the excluder doesn't have a CesiumGeoreference at all, or if its different from the Tileset's CesiumGeoreference, then a message will be written to the log and the excluder will be ignored.